src/containers/trends/SMBGRangeAvgAnimationContainer.js

```diff
@@ -105,21 +106,28 @@ export default class SMBGRangeAvgAnimationContainer extends React.Component {
     const yPositions = this.calcYPositions(
       mungedData, yScale, (d) => (spring(yScale(d)))
     );
-
     return (
       <TransitionMotion styles={yPositions}>
         {(interpolated) => (
           <g id="smbgRangeAvgAnimationContainer">
             {_.map(interpolated, (config) => (
```

Comment:
Not necessary now, but eventually we will have to split this into two `{_.map(interpolated, ...)}` because elements that are not animated with the same interpolations will be going in between—i.e., the individual smbgs layered by date.